app/assets/javascripts/osem.js

```diff
@@ -184,8 +184,9 @@ $( document ).ready(function() {
         $("#abstract-minimum-word-count").text(min);
         $("#submission-minimum-word-count").text(min);
         word_count($('#event_abstract').get(0), 'abstract-count', max);
-        word_count($('#event_submission_text').get(0), 'submission-count', max);
-    }).trigger('change');
+    }
+    $("#event_event_type_id").change(updateEventTypeRequirements);
```

Comment:
Not sure if we should actually keep the word count for the `#event_submission_text` field.